server.py

```diff
@@ -42,15 +42,15 @@ def register_user():
     #Alert if email already exists
     if crud.get_user(email):
         flash('This email already exists. Please login')
-    
+
     #Otherwise, create new user account, add to session, and redirect to profile
-    else: 
-        user = crud.create_user(email, username, first_name, last_name, password)
```

Comment:
We talked about this during our meeting. You don't need to assign this to a variable